app/src/main/java/ch/epfl/skysync/components/ConfirmFlightUI.kt

```diff
@@ -41,7 +44,11 @@ import java.time.format.DateTimeFormatter
  * @param plannedFlight The planned flight for which the confirmation screen is displayed.
  */
 @Composable
-fun confirmationScreen(plannedFlight: PlannedFlight) {
+fun confirmationScreen(
+    plannedFlight: PlannedFlight,
+    navController: NavController,
```

Comment:
do you use navController in this Composable? If yes, try to keep the navConstorller only in the toplevel Composable and work with onClick lambdas e.g.
if you use it for the "back" button define a onBackClick lambda

Reply:
fixed